src/pipeline/common/Manifest.groovy

```diff
@@ -99,6 +99,23 @@ def generateManifestFromPR( String pr_url, String pr_branch, String dest_manifes
 }
 
 
+def generateManifestFromPR(String manifest_path, String github_repository, String commit){
```

Comment:
actually, it's generateManifestFromGHPRB() :-).............
because there was a ```generateManifestFromPR()```

Reply:
Thanks, fixed!